packages/types/src/global.ts

```diff
@@ -5,9 +5,12 @@ export type ComponentExtras = {
     dirname: string;
     basename: string;
 }
+
+export type GroupingPath = { name: string, type: 'GROUP' | 'FRAME' }
 export interface ComponentNode extends Figma.Component {
     figmaExport: ComponentExtras;
     svg: string;
+    groupingPath: GroupingPath[];
```

Comment:
I would move the `groupingPath` property into `ComponentExtras` to avoid possible future naming collision with `Figma.Component`. This is also useful in all official outputters since ComponentExtras is part of `ComponentOutputterParamOption`.

Reply:
Do you have an example of the output for `containing_frame`? I don't have a paid account so I cannot use that API.

I feel that `containing_frame` and `groupingPath` are two different things since the `groupingPath` is an array of paths to reach the selected component, while `containing_frame` is a plain object (`FrameInfo`).

I was reflecting on the `groupingPath` name, another valid name could be `pathToComponent`. The fact is that `group` is a node type, and having a variable containing that name could confuse someone.
`pathToComponent` is clear and concise, and it accurately reflects the purpose of the variable.

Reply:
FYI, this is the signature of `containing_frame` FrameType ([link](https://www.figma.com/developers/api#frameinfo-type)).

Ok, I agree with you, applied changes already :D